python/cuml/__init__.py

```diff
@@ -165,6 +176,7 @@ def __getattr__(name):
     "TreeExplainer",
     "TSNE",
     "UMAP",
+    "UniversalBase",
```

Comment:
I'm on the fence on that one. I think it might be useful for external libraries trying to integrate cuML in a CPU/GPU codebase, but I don't think it's absolutely necessary.